doc/examples/datetimes.rst

```diff
@@ -25,10 +25,12 @@ time into MongoDB:
 
 .. doctest::
 
-   >>> result = db.objects.insert_one({"last_modified": datetime.datetime.utcnow()})
+   >>> result = db.objects.insert_one(
+   ...     {"last_modified": datetime.datetime.now(tz=timezone.utc)}
+   ... )
 
-Always use :meth:`datetime.datetime.utcnow`, which returns the current time in
-UTC, instead of :meth:`datetime.datetime.now`, which returns the current local
```

Comment:
Hmm would it be worth adding a note in the changelog about this? Like "Previously PyMongo's docs recommended using utcnow. utcnow is deprecated in Python 3.12. Instead , we now recommend using X."

I'm thinking we can help point users to the right workaround since pymongo apps will likely have copied this pattern of using utcnow and will start seeing deprecation warnings in 3.12.

Reply:
Agreed, I imagine a lot of users will start seeing deprecation warnings in 3.12. Adding a note in the changelog with a link to the Python PR change explanation is a good idea.